jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/PropertyRule.java

```diff
@@ -16,6 +16,7 @@
 
 package org.jsonschema2pojo.rules;
 
+import jakarta.validation.constraints.Email;
```

Comment:
Nitpick: currently this is unused import, since fully-qualified class name is used on L207.